pkgs/tools/misc/esphome/default.nix

```diff
@@ -19,14 +19,14 @@ let
 in
 python.pkgs.buildPythonApplication rec {
   pname = "esphome";
-  version = "2024.4.2";
+  version = "2024.5.0";
```

Comment:
We should probably straight go to 2024.5.1 https://github.com/esphome/esphome/releases/tag/2024.5.1

Reply:
We will, but this release contains breaking changes, so it has to wait.